lib/rules/forbid-prop-types.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
please remove this

Reply:
Fixed in[25e6afd](https://github.com/jsx-eslint/eslint-plugin-react/pull/3731/commits/25e6afd2a02e62af7d806fa6c2e12de46a97dcad).